docs/docs/services/licenses.md

```diff
@@ -35,28 +38,39 @@ support multiple instances if needed.
 
 ### Health check
 
-For health checking, the service exposes `/v1/health` on port 9200, and it always responds with `200 OK`.
+For health checking, the service exposes `/health` on port 9200, and it always responds with:
+
+```json
```

Comment:
Maybe it's better to just write that the licenses service returns status code `200`  if alive. Not sure how the api spec for the licenses service should be handled in the long run, wether it will be public or not.